### DISCUSSION TITLE
Failed to load module script

### DISCUSSION THREAD

**Discussion by Arthur-Chevron** [nuxt/nuxt#17066] [Questions]
Hello, 

i have this issue for my nuxt3 application :
"Failed to load module script: The server responded with a non-JavaScript MIME type of “application/octet-stream”. Strict MIME type checking is enforced for module scripts per HTML spec."

So any js can be execute ?
Any idea ?

**Comment by alex-maereanu** [comment]
Same here .. have you found a resolution?

**Reply by iku8** [reply]
@alex-maereanu 
Do you use nginx?

It works well if you write the following settings.
```
    location ~* \.mjs$ {
        types {
            application/javascript mjs;
        }
    } 
```

This is same issue.
https://github.com/nuxt/framework/issues/5303

**Reply by alex-maereanu** [reply]
Thanks @iku8!

**Comment by Silbad** [comment]
Hello, and if I can't change the server configuration, what can I do? Can we build in simple js files?
https://github.com/GeoffreyBooth/js-mjs-mime-type-test

**Comment by arthurducept** [comment]
Same error here :/
I've found this blog ([link](https://kas.kim/blog/failed-to-load-module-script)) that refers to the exact same error message i got, but i can't find a solution.
I'm using Nuxt 3, with SSR generation and nginx to render the pages on my server.

**Comment by danielroe** [comment]
Just so you know, the edge channel is using .js as output file extension. You can opt in following these steps: https://v3.nuxtjs.org/guide/going-further/edge-channel/
